docs/platforms.md

```diff
@@ -241,6 +314,6 @@ If your project requires additional tools to build (such as `cmake`, `ninja`, or
 
 If tests have been configured, the test suite will be executed on the simulator matching the architecture of the build machine - that is, if you're building on an ARM64 macOS machine, the ARM64 wheel will be tested on an ARM64 simulator. It is not possible to use cibuildwheel to test wheels on other simulators, or on physical devices.
 
-The iOS test environment can't support running shell scripts, so the [`test-command`](options.md#test-command) value must be specified as if it were a command line being passed to `python -m ...`. In addition, the project must use [`test-sources`](options.md#test-sources) to specify the minimum subset of files that should be copied to the test environment. This is because the test must be run "on device", and the simulator device will not have access to the local project directory.
+The iOS test environment can't support running shell scripts, so the [`test-command`](options.md#test-command) value must be specified as if it were a command line being passed to `python -m ...`.
```

Comment:
This change is to the iOS docs, it looks unintentional to me?

Reply:
This is intentional: the requirement to always use `test-sources` on iOS was removed in #2420, but the documentation wasn't updated.

Reply:
Well, it's still required if the user wants to use test files from the project dir. So I think it's still accurate to say that you must use it to "specify the minimum subset of files" - In #2420, we changed the logic so that if that happens to be nothing, you can leave it unset.